### PR TITLE
Add centos/7.5/aws/DCOS-1.13/docker-18.09.2-ce/selinux_disabled

### PR DESCRIPTION
**Pull request by cprovencher**
- pin terraform version again
- update terraform-dcos, which means stripping a lot of the old code that adds an OS bcause we can now use the aws_ami parameter
- add cloudcleaner tags to clusters
- add nvme/rexray support
- add a systemd service that mounts tmp noexec,nosuid,nodev at boot
- add dss modules raid1 and dmraid
- removing setup.sh as we want to build everything directly in the AMI. Removing setup.sh was made possible by my terraform-dcos changes. https://github.com/dcos/terraform-dcos/pull/75
- removing tests_to_run from the publish_and_test_config.yaml. This was a workaround while test_mom_installation was broken.

### CODE REVIEW

**Comment by cprovencher** [comment]
running here: https://teamcity.mesosphere.io/viewLog.html?tab=buildLog&buildTypeId=DcOs_Open_Test_IntegrationTest_AwsOnpremWStaticBackend&buildId=1984403

**Comment by ahairape** [comment]
@cprovencher do you have any idea what might have caused the failure: https://jenkins.mesosphere.com/service/jenkins/blue/organizations/jenkins/dcos-images/detail/PR-232/31/pipeline#step-93-log-797 ?

**Comment by cprovencher** [comment]
> @cprovencher do you have any idea what might have caused the failure: https://jenkins.mesosphere.com/service/jenkins/blue/organizations/jenkins/dcos-images/detail/PR-232/31/pipeline#step-93-log-797 ?

@ahairape There is an issue right now where it seems like machines are getting killed in the middle of integration tests. I have tested with dcos-launch on a different account, using the same AMIs and haven't gotten the issue. https://jira.mesosphere.com/browse/DCOS-55342

**Comment by ahairape** [comment]
@cprovencher The `continuous-integration/jenkins/pr-head` check had passed before adding this commit: https://github.com/dcos/dcos-images/pull/232/commits/c09ce6d750b477a6548bd811341282db50e2ebbf
Could bumping `terraform-dcos` cause the failure?

File: Jenkinsfile [hunk 1]
```diff
@@ -74,8 +74,8 @@ node('mesos-ubuntu') {
   }
 
   stage("Get packer") {
-    shcmd("""apt-get install -y curl &&
-          curl -L -O https://releases.hashicorp.com/packer/1.2.4/packer_1.2.4_linux_amd64.zip &&
+    shcmd("""apt-get install -y wget &&
```
Comment (orsenthil) [inline Jenkinsfile]: btw, this is fine!. but most often curl is more commonly available than wget. The retries mechanism in curl are better and more feature rich tha wget.

That's why in dcos we use curl a lot.

**Comment by orsenthil** [comment]
@cprovencher  - why is this CI failing?

**Comment by orsenthil** [comment]
It seems a certain integration test is failing.

**Comment by cprovencher** [comment]
@ahairape The reason it started failing after that commit is because I specified to run all the tests instead of the two that were previously specified. More tests take longer and then the "connection closed by remote host" occurs

**Pull request merged**